crates/avalanche-types/tests/rpc/common.rs

```diff
@@ -58,17 +60,13 @@ pub fn generate_http_request(
     address: &str,
     param: &[&str],
 ) -> http::Request<Vec<u8>> {
-    let mut json_params = Vec::with_capacity(param.len());
+    let raw_params = serde_json::value::to_raw_value(param).unwrap();
 
-    for i in 0..param.len() {
-        json_params.push(serde_json::Value::from(param[i]))
-    }
-
-    let m = jsonrpc_core::MethodCall {
-        jsonrpc: Some(jsonrpc_core::Version::V2),
-        method: String::from(method_name),
-        params: jsonrpc_core::Params::Array(json_params),
-        id: jsonrpc_core::Id::Num(1),
+    let m = Request {
+        jsonrpc: TwoPointZero,
+        method: Cow::Borrowed(method_name).into(),
```

Comment:
nit:
```suggestion
        method: method_name.into(),
```

Reply:
Ignore this too... but just FYI, no need to convert from `&str` to `std::borrow::Cow` and then to `beef::Cow`. I'm sure the assembly would have been the same, but 🤷